### ISSUE TITLE
Experimental problem

### ISSUE THREAD

**Issue by zhiwudazhanjiangshi123123** [deep-geo/MoE-NuSeg#1]
Hello, thank you for your paper and code. I have a question and hope to get your help. When I was reproducing the experiments, the effects of many models exceeded the expected results. Could you provide some configurations for the experiments? Thank you for your reply.

**Comment by deep-geo** [comment]
Thanks  for your interest in our work and for sharing your reproduction results!

We’re glad to hear that some results are exceeding expectations — that’s exciting. To help align with our official configuration for Phase 1 and Phase 2 (with router), here are the typical experimental settings we used:

	•	Model: MoE_p1
	•	Loss Function: 0.4 * CE + 0.6 * Dice for each of the three outputs (nuclei, normal edge, cluster edge)
	•	Input Channels: 3-channel RGB
	•	**Random Seed: commonly between 41–46**..  THIS MIGHT BE THE CAUSE
	•	Batch Size: 2 
	•	Learning Rate: 1e-4 with CosineAnnealingLR
	•	Epochs: ~400 with early stopping

Sample config: 
  --alpha=0.3 \
  --beta=0.35 \
  --gamma=0.35 \
  --dataset=Histology \
  --num_epoch=400 \
  --lr=0.001 \
  --random_seed=41 \  (from 41-46)
  --batch_size=2

This configuration trains the router and full MoE architecture on the Histology dataset, with 3 output branches and explicit loss balancing across segmentation and edge detection tasks.


Best regards,
Xuening Wu
